src/main/resources/db/migration/V42_3__Create_user_table.sql

```diff
@@ -0,0 +1,14 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+
```

Comment:
start the name of migration file by "V0_1__ ....."

Reply:
That's a great remark.
But for this one, the project was based on the POJA library, then there was already a generated "V42_1__Create_dummy_table.sql" at beginning of the migration, that's why I had to follow the sequence.

Reply:
you are rigth